docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -93,11 +93,28 @@ export S3_ENDPOINT_URL='https://<accountid>.r2.cloudflarestorage.com'
 
 ## Google Cloud Storage
 
-### MosaicML platform
+### GCP Service Account Credentials
+
+#### MosaicML platform
```

Comment:
I think it would be better to move the `MosaicML platform` above `GCP Service Account Credentials` ? since it will be same for HMAC and service level authenticaltion.